pkg/kubectl/cmd/get/get.go

```diff
@@ -638,78 +640,77 @@ func (o *GetOptions) watch(f cmdutil.Factory, cmd *cobra.Command, args []string)
 		return err
 	}
 
-	// watching from resourceVersion 0, starts the watch at ~now and
-	// will return an initial watch event.  Starting form ~now, rather
-	// the rv of the object will insure that we start the watch from
-	// inside the watch window, which the rv of the object might not be.
-	rv := "0"
```

Comment:
fyi @lavalamp @wojtek-t @liggitt one `RV "0"` down

Reply:
for others we should never use watch calls from `RV "0"`, see e.g. https://github.com/kubernetes/kubernetes/issues/74022